src/Maybe.js

```diff
@@ -11,9 +11,15 @@ function _Just(x) {
 }
 util.extend(_Just, Maybe);
 
+_Just.prototype.isJust = R.T;
+_Just.prototype.isNothing = R.F;
```

Comment:
Would work as well. But I think it's more intuitive when `isJust`, `isNothing`, `isLeft` and `isRight` are functions.
But I could live with both.